wcivf/apps/elections/templates/elections/includes/_cancelled_election.html

```diff
@@ -1,27 +1,81 @@
 {% load markdown_deux_tags %}
+{% load humanize %}
+{% load static %}
+
 {% if object.metadata.cancelled_election.title %}
     <h2>{{ object.metadata.cancelled_election.title }}</h2>
 {% else %}
-    <h2>Cancelled Election</h2>
+    <div>
+        {% if object.winner_count == object.people.count %}
+            <b>Uncontested Election</b>
```

Comment:
A note on the `b` tag. It's got a bit of a complex history, where it was used for "bold" in the early days, then not used at all ("strong" replaced it) and now it's back in a different form to mean "bring to attention". 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

I can see the case for using "bring to attention" here, but the above link suggests not using it for headings. 

Because of that, I thin this should be a heading under the heading level of the ballot title (`h4`?). If the size looks wrong, we can use the `.ds-h[n]` classes to change that.

Reply:
Ballot name is `h2` but this also made me realise that I need to move the ballot name out of the conditional block.